pkg/proc/native/proc_linux.go

```diff
@@ -689,6 +706,46 @@ func (dbp *nativeProcess) EntryPoint() (uint64, error) {
 	return linutil.EntryPointFromAuxv(auxvbuf, dbp.bi.Arch.PtrSize()), nil
 }
 
+func (dbp *nativeProcess) SupportsBPF() bool {
+	return true
```

Comment:
This return value should depend on the ebpf tag.

Reply:
Done.